src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -34,4 +45,15 @@ public Command parseCommand(String userInput) {
             return new HelpCommand();
         }
     }
+
+    private Command prepareAdd(String arguments) {
+
+        final int indexOfAtPrefix = arguments.indexOf("i/");
+        final int indexOfAtPrefix_2 = arguments.indexOf("p/");
```

Comment:
As I have mentioned in the telegram chat, it is better that you include all 3 cases when parsing this command:
ADD i/ITEM
ADD i/ITEM p/PRICE
ADD p/PRICE i/ITEM

Reply:
Noted! Thanks ^^